util/artifacts_tar.go

```diff
@@ -106,7 +106,7 @@ func BuildArchive(ctx context.Context, tarWriter *tar.Writer, rootPath string, i
 				errChan <- errors.Wrapf(err, "Error opening %v", file.Path)
 				return
 			}
-
+			logger.Debugf("Beginning copy (header size %v)\n", hdr.Size)
```

Comment:
I'm worries about adding logging in a for loop. The agent could be tarring up a giant archive. Would a log statement right after the loop be useful?